tests/test_models.py

```diff
@@ -10,6 +10,7 @@
 import logging
 import pytest
 import asyncio
+import os
```

Comment:
Looks like the import is unused.

Reply:
Ah just spotted this, I'll add one additional commit to just remove this non-required dependency as we are going to rebase anyhow.